src/test/regress/upgrade/generate_citus_tarballs.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+
+citus_versions=( $1 master )
+
+base="$(pwd)"
+
+build_ext() {
+  citus_version="$1"
+  basedir="${base}/${citus_version}"
+  
+  mkdir -p ${basedir}
+  cd ${basedir}
+  citus_dir=${basedir}/citus_$citus_version
+  git clone --branch $citus_version https://github.com/citusdata/citus.git --depth 1 citus_$citus_version  
+  builddir="${basedir}/build"
+
+  # do everything in a subdirectory to avoid clutter in current directory
+  mkdir -p "${builddir}" && cd "${builddir}"
+
+  "${citus_dir}/configure" --without-libcurl
+
+  installdir="${builddir}/install"
+  make && mkdir -p "${installdir}" && make DESTDIR="${installdir}" install
```

Comment:
Let's use those cores
```suggestion
  make "-j$(nproc)" && mkdir -p "${installdir}" && make DESTDIR="${installdir}" install
```